portfolio/.gitignore

```diff
@@ -0,0 +1 @@
+portfolio/target
```

Comment:
this is the gitignore file for the portfolio directory, so I dont think you need this if you added it to the main gitignore dir above